### PR TITLE
Number System

### LINKED ISSUE #6765: Add number system in Python
### Description

What is a number system?
What are the types of number systems in Python with examples?

### Domain

Python_Domain

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by priyankapiba**
<hr>

## Description 📜

Number system, i.e., binary number system, decimal number system and many more, explanation with examples as well as python program for converting from one form to another.

Fixes #6765  

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [x] Python_Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@priyankapiba please make these changes

File: Python/Number System/Conversion Programs/Decimal-to-other-base-conversion.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+if __name__ == '__main__':
+     
+    # decimal value
+    dec_val = int(input())
+     
+    # Calling function
+    DecimalToBinary(dec_val)
+```
+## OUTPUT  
+24 <br>
```
[the remark this answers is not in the document]
Comment (priyankapiba) [reply Python/Number System/Conversion Programs/Decimal-to-other-base-conversion.md:27]: I didn't get this
Can you please explain this

[the remark this answers is not in the document]
Comment (apeksha235) [reply Python/Number System/Conversion Programs/Decimal-to-other-base-conversion.md:27]: @priyankapiba here's an example: 
Input 
```
24
``` 
Output 
```
011000
```

[the remark this answers is not in the document]
Comment (priyankapiba) [reply Python/Number System/Conversion Programs/Decimal-to-other-base-conversion.md:27]: Ok done ma'am.

File: Python/Number System/num_sys.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ ## 3. Decimal number system
+  This system of numbers we use in our daily life calculation . in this system we can represents any number with the help of 10 symbols from 0 to 9
+ ## 4. Hexadecimal number system
+This system is a base 16 system which uses symbols from 0 to 9  and A to F in which 'A' has positional weight 10, 'B' has positional weight 11 and so on till 16 for 'F'.<br>
+ - To convert from **hexadecimal to decimal** we have to multiply each digit with  the powers of 16 of their respective positional weight. For example:-<br>
+ (20A.2C)<sub>8</sub> <br>
+   2x16<sup>2</sup> + 0x16<sup>1</sup>+ 10x16<sup>0</sup> + 2x16<sup>-1</sup> + 12x16<sup>-2</sup> =512 + 0 + 10 + 0.171878= 522.171878 <br>
+  (20A.2C)<sub>16</sub> = (522.171878)<sub>10</sub> <br>
+  -To convert **decimal to hexadecimal** we have to repetedly divide the number by 16 and print the remainder in reverse order .for example:-<br>
+  click the [link](https://circuitglobe.com/wp-content/uploads/2016/09/deciaml-to-hexadecimal-conversion-6.jpg).
```
Comment (apeksha235) [inline Python/Number System/num_sys.md:38]: @priyankapiba you can directly copy paste the image on md file instead of adding a link

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@priyankapiba

File: Python/Number System/num_sys.md (hunk 2 above)
Comment (priyankapiba) [reply Python/Number System/num_sys.md]: done @apeksha235

**Comment by apeksha235** [comment]
Good! @priyankapiba 
@adityabisoi Can be merged!

**Pull request merged**